### DISCUSSION TITLE
Keycloak Grant Acess page customization

### DISCUSSION THREAD

**Discussion by tiagogalvao7** [keycloak/keycloak#32028] [Q&A]
I'm developing a theme for Keycloak, and I've already been able to customize the login page, the page to create a new user, but I still haven't managed to customize the "grant access" page. Can you explain how I can do this? Supposedly the Login theme allows you to change the Grant access page. I already have the layout created, but I haven't been able to apply it to Keycloak yet. I'm talking about the page in the image below:
![image](https://github.com/user-attachments/assets/b69952de-1161-4562-b73e-f00c4464eed8)
![image](https://github.com/user-attachments/assets/51bffbef-722f-405b-80c7-534681aabd67)

**Comment by xXAvoraXx** [comment]
Is it [Keycloakify](https://docs.keycloakify.dev/) ?
